README.md

```diff
@@ -2,3 +2,35 @@ Cloud Marker
 ============
 
 Cloud Marker is a cloud security monitoring framework.
+
+
+Setup Development Environment
+-----------------------------
+
+Please follow these steps to setup the development environment:
+
+ 1. Ensure Python 3 is installed.
+
+        # On macOS
+        brew install python
+
+ 2. Clone the project repository.
+
+        git clone https://github.com/cloudmarker/cloudmarker.git
+
```

Comment:
You might want to add the make command `make venv` and `. ./venv` here for setting up the project.